persistence/persistence-mybatis-test/src/test/resources/testContextDatabase.xml

```diff
@@ -5,17 +5,15 @@
 	xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
Why does this file have any changes to it as compared to the code checked into the RC branch:
https://github.com/cBioPortal/cbioportal/blob/rc/persistence/persistence-mybatis-test/src/test/resources/testContextDatabase.xml